src/commands/ext-install.ts

```diff
@@ -112,21 +114,68 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
 }
 
 /**
- * Command for installing a extension
+ * Command for installing an extension
  */
-export default new Command("ext:install <extensionName>")
-  .description("install an extension, given <extensionName> or <extensionName@versionNumber>")
+export default new Command("ext:install [extensionName]")
+  .description(
+    "install an extension if [extensionName] or [extensionName@version] is provided; or run with `-i` to see all available extensions."
+  )
   .option("--params <paramsFile>", "name of params variables file with .env format.")
   .before(requirePermissions, ["firebasemods.instances.create"])
   .before(ensureExtensionsApiEnabled)
   .action(async (extensionName: string, options: any) => {
+    const projectId = getProjectId(options, false);
+    const paramFilePath = options.params;
+    let learnMore = false;
+    if (!extensionName) {
+      if (options.interactive) {
+        learnMore = true;
+        extensionName = await promptForOfficialExtension(
+          "Which official extension would you like to install?\n" +
+            "  Select an extension, then press Enter to learn more."
+        );
+      } else {
+        throw new FirebaseError(
+          `Please provide an extension name, or run ${clc.bold(
+            "firebase ext:install -i"
+          )} to see all available official extensions`
```

Comment:
creating parity with error below:

to select from the list of all available official extensions.